src/PFXImportPowershell/OnPremValidation/IntunePFXCertConnectorValidationPS/InstallAndRunTestService.ps1

```diff
@@ -1,5 +1,5 @@
 param(
-	[string] $serviceDirectory,
+	[string] $serviceDirectory = "..\IntunePFXCertConnectorValidationService\bin\x64\Debug\",
```

Comment:
Is this validation only for debug builds?

Reply:
This is a default value. They can override it if they do not want to do a debug build.